documentation/configure.markdown

```diff
@@ -387,6 +387,18 @@ runs every `gc-interval` minutes.
 
 If unset, the default value is 14 days.
 
+### `resource-events-ttl`
+
+Automatically delete report events older than the specified time. The reports
+will still be available, but the contained events will not be present.
+
+This allows for more fine-grained control of the expiration of reports. If this
+value is set higher than `report-ttl`, it will have no effect.
```

Comment:
We don't seem to handle the case where report-ttl is less than resource-events-ttl directly, does report gc always happen when a resource event gc happens? If you can trigger them separately, we probably want to overwrite the resource-event-ttl setting if it is greater than the report-ttl setting.

Reply:
No, it's the other way around - a report gc implies a resource-events gc. It's really harmless if it's larger than report gc - what'll happen is a delete query that doesn't do anything.